go-kit/metrics/provider/librato/librato_test.go

```diff
@@ -341,7 +341,7 @@ func TestLibratoSingleReportWithLabelValuesOnTagBasedAccount(t *testing.T) {
 		t.Fatal("unexpected error reporting metrics", err)
 	}
 
-	p := New(u, doesntmatter, WithTags(), WithSource("test.source"), WithErrorHandler(errs))
+	p := New(u, doesntmatter, WithTags("app", "myapp"), WithSource("test.source"), WithErrorHandler(errs))
```

Comment:
It'd be great to test that the tags show up?

Reply:
+1